noether_filtering/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ find_package(console_bridge REQUIRED)
 find_package(pluginlib REQUIRED)
 find_package(xmlrpcpp REQUIRED)
 
+# Check if PCL was compiled with OpenNURBS
+set(NURBS_FOUND TRUE)
+
```

Comment:
This should be removed? I think this is why the noetic build without docker is failing.

Reply:
Yeah. I took it out to see if that was causing the issue.

Reply:
OK, I added it back in and now at least noetic is passing. I working on the docker for melodic now.